### PR TITLE
Deprecate listPrice and manufacturerPrice in storefront events

### PR DESCRIPTION
**Pull request by depoulo**
EPUI-1830

### CODE REVIEW

File: apps/change-log.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2023-0?-??
```
Comment (depoulo) [inline apps/change-log.md:17]: @ePages-de/team-emerald FYI

File: apps/change-log.md [hunk 2]
```diff
@@ -14,6 +14,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
...
+
+### Changes with software release 7.?.?
+
+#### <i class="fa fa-minus"></i> Deprecate
+
+* The properties `listPrice` and `manufacturerPrice` in the `product` and `products` properties of the [events](https://developer.epages.com/apps/events.html) `product:view`, `product:click`, `category:view`, `searchResults:view`, and `cart:add` are deprecated and will be removed on ? ?th, 2023. As a replacement, the new properties `strikePrice` and `isStrikePriceRRP` have been added.
+
```
Comment (depoulo) [inline apps/change-log.md:24]: FYI @ePages-de/team-emerald due to the release cycle, I left out the dates, for you to update later.

**Review by Aliamondo — APPROVED** [review]

File: apps/change-log.md (hunk 1 above)
Comment (dkluever22) [inline apps/change-log.md:17]: I would say here we use the dates for the upcoming roll out of 7.80 to announce the deprecation

```suggestion
## 2023-02-16
```

**Review by dkluever22 — CHANGES_REQUESTED** [review]
If you agree to the changes I suggested, it would be important to wait until Thursday, February 16th, before merging this PR.

**Review by dkluever22 — APPROVED** [review]

**Pull request merged**